doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -222,6 +222,97 @@ gls{Palladio}
  runs on.
 \end_layout
 
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ20/"
+
+\end_inset
+
+ The measurements run without user interaction.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ30/"
+
+\end_inset
+
+ The measurements also shall run over the network e.g.
+ on a Server.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ40/"
+
+\end_inset
+
+ The user has the possibility to pause the measurements.
+ As well the user can continue a measurement from save.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ50/"
+
+\end_inset
+
+ If it is requested, Beagle shuts down the PC after it finished.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ60/"
+
+\end_inset
+
+ Beagle report the progress of measurements in a progress bar.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ70/"
+
+\end_inset
+
+ Beagle can handle errors e.g.
+ exceptions.
+\end_layout
+
+\begin_layout Labeling
+\labelwidthstring 00.00.0000
+\begin_inset CommandInset label
+LatexCommand label
+name "/OQ80/"
+
+\end_inset
+
+ Beagle stops measruements by timeout.
```

Comment:
We wrote about something regarding an adaptive timeout. This should get into the Requirements Specification. If it currently isn't you can add it.